editor/backend/src/EditCommand.ts

```diff
@@ -15,7 +15,21 @@ import { EditCommandIpc, EditorIpc, editorIpcStrings } from "@itwin/editor-commo
 export type EditCommandType = typeof EditCommand;
 
 /**
- * An EditCommand performs an editing action on the backend. EditCommands are usually paired with and driven by EditTools on the frontend.
+ * An EditCommand performs an editing action on the backend.
+ * Any writes to the iModel in an editing session should be done from an EditCommand:
+ * - All changes to an iModel are made within a transaction (Txn).
+ * - Calling SaveChanges ends the current Txn and starts a new one.
+ * - Using EditCommand ensures all of the changes in a Txn are from the same source, as only one EditCommand may be active at a time.
+ * - Because there is currently no way to enforce this, it is important that all applications follow this rule.
```

Comment:
Is there an example EditCommand we can link to?

Reply:
Only one, I could add "@see BasicManipulationCommand for an example" if you think that helps?

Reply:
Sure.

Reply:
done.